Gemfile

```diff
@@ -48,6 +48,7 @@ gem 'sass-rails'
 # Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
 
+gem 'webpack-rails'
```

Comment:
Was asking cause webpacker would be used in Rails 5.1 - but if webpacker only supports javascipt, it might not be a good solution for us too. We can revisit this again next time if need be. 

Switching between either shouldn't be too hard either so I think that works. My only concern is whether the gem will be maintained (versus webpacker, which has some guarantee as long as rails is in development), but it seems to be actively maintained :+1: